datetime/datetime.go

```diff
@@ -350,6 +350,11 @@ func datetimeDecoder(d *msgpack.Decoder, v reflect.Value, extLen int) error {
 	return ptr.UnmarshalMsgpack(b)
 }
 
+// This method converts Datetime to String - formats to ISO8601.
```

Comment:
The go-idiomatic way to comment a method.
```suggestion
// String converts Datetime to String - formats to ISO8601.
```